var/spack/repos/builtin/packages/opium/package.py

```diff
@@ -16,6 +16,7 @@ class Opium(Package):
 
     depends_on('blas')
     depends_on('lapack')
+    build_targets = ['clean']
 
     def install(self, spec, prefix):
         libs = spec['lapack'].libs + spec['blas'].libs
```

Comment:
I don't think we want to run configure twice. These options should go in the `configure_args` phase. The build stuff should be moved to the build phase.

Reply:
I divided it into three phases.